apps/src/sites/studio/pages/levels/_pixelation.js

```diff
@@ -728,19 +725,6 @@ function changeVal(elementID) {
 
 window.changeVal = changeVal;
 
-function setSliders() {
```

Comment:
how do we know that this isn't used?

Reply:
Several redundant ways to verify -
- `eslint` itself is complaining about it being unused, which was the reason I dropped it.
- The function itself isn't exported from the file, so it's only accessible within it. A search in the file confirms that it is not used within it.
- Grepping the codebase confirms the only instance is the definition.

Very safe to get rid of.